package.json

```
@@ -91,6 +91,59 @@
             }
           }
         }
+      },
+      {
+        "name": "apply_projectTemplate",
+        "when": "confluent.chatParticipantEnabled",
+        "tags": [
+          "projects"
+        ],
+        "displayName": "Apply Project Template",
+        "modelDescription": "Apply a selected project template to generate a streaming data project or application. Use this when the user selects a template and provides the necessary options.",
+        "canBeReferencedInPrompt": false,
+        "icon": "$(checklist)",
+        "inputSchema": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "The name of the template to apply."
+            },
+            "options": {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              },
+              "description": "Key-value pairs of options to configure the template."
+            }
+          },
+          "required": [
+            "name"
+          ]
+        }
+      },
+      {
+        "name": "get_projectOptions",
+        "when": "confluent.chatParticipantEnabled",
+        "tags": [
+          "projects"
+        ],
+        "displayName": "Get Project Options",
+        "modelDescription": "Provides the necessary options for calling apply_projectTemplate. Use this when the user selects a template and provides the necessary options.",
```

What do you think of this wording?
```suggestion
        "modelDescription": "The key-value pairs used in order to generate a project from a selected template.",
```
Mainly concerned about this part: `Use this when the user [...] provides the necessary options.`